spec/services/exports/case_log_export_service_spec.rb

```diff
@@ -16,9 +16,8 @@
 
   def replace_entity_ids(case_log, export_template)
     export_template.sub!(/\{id\}/, (case_log["id"] + Exports::CaseLogExportService::LOG_ID_OFFSET).to_s)
-    export_template.sub!(/\{owning_org_id\}/, case_log["owning_organisation_id"].to_s)
-    export_template.sub!(/\{managing_org_id\}/, case_log["managing_organisation_id"].to_s)
-    export_template.sub!(/\{created_by_id\}/, case_log["created_by_id"].to_s)
+    export_template.sub!(/\{owning_org_id\}/, (case_log["owning_organisation_id"] + Exports::CaseLogExportService::LOG_ID_OFFSET).to_s)
```

Comment:
`is owning_org_id` and `case_log["owning_organisation_id"]` different values?

Reply:
`{is owning_org_id}` is just a placeholder inside our CSV and XML examples that we replace with the owning organisation ID we get when Factory Bot initialises the case log (and related organisations).

Reply:
Oh, of course 🤦‍♀️  tty